packages/solid/components/Badge/Badge.styles.ts

```diff
@@ -74,7 +68,10 @@ const text: BadgeConfig = {
   },
   base: {
     ...theme.typography.tag1,
-    color: theme.color.textNeutral
+    color: theme.color.textNeutral,
+    lineHeight: theme.typography.tag1.lineHeight + 12,
+    marginRight: theme.spacer.md + theme.stroke.sm,
+    marginLeft: theme.spacer.md + theme.stroke.sm
```

Comment:
it looks like we removed padding rules in favor of marginLeft and marginRight - are these interchangeable, are the margin rules related to flex or something else?

Reply:
Padding doesn't exist in the flex layout. MarginRight & MarginLeft do and will do the same thing...

Reply:
padding rules work withPadding - but again would like to deprecate withPadding in favor of using flex with margins